paragon/_overrides.scss

```diff
@@ -1,6 +1,10 @@
 // Use this file to define style overrides for @edx/paragon
 // This file is included after all Paragon styles, but should generally avoid using private mixins in Paragon.
 
+.btn {
+    border-radius: 3em;
```

Comment:
@sophiazumich Did you try defining a value for [`$btn-border-radius`](https://paragon-openedx.netlify.app/components/button/#theme-variables-scss) in `_variables.scss`? If that works I think it could be preferable to adding an override in this file.

Reply:
@michaelwheeler @aphilipharrell I was able to make the button radius update with this variable and it looks right on my local Quail Page! Hopefully the buttons still look pill-shaped on the delete modal branch/partner portal env.